bin/run-addon

```diff
@@ -92,15 +102,15 @@ if [[ -n "$setup_profile" ]] ; then
   profile_dir="$(pwd)/Profile"
   if [[ -d "$profile_dir" ]] ; then
     echo "Using existing profile in $profile_dir running:"
-    echo " " open "$binary" -n --args -new-instance -no-remote -profile "$profile_dir"
-    open "$binary" -n --args -new-instance -no-remote -profile "$profile_dir"
+    echo " " $open -n --args -new-instance -no-remote -profile "$profile_dir"
+    $open -n --args -new-instance -no-remote -profile "$profile_dir"
   else
     echo "Creating profile in $profile_dir running:"
-    echo " " open "$binary" -n --args -new-instance -no-remote -CreateProfile "PageShotTestProfile $profile_dir"
-    open "$binary" -n --args -new-instance -no-remote -CreateProfile "PageShotTestProfile $profile_dir"
+    echo " " $open -n --args -new-instance -no-remote -CreateProfile "PageShotTestProfile $profile_dir"
+    $open -n --args -new-instance -no-remote -CreateProfile "PageShotTestProfile $profile_dir"
     echo "Opening profile running:"
-    echo " " open "$binary" -n --args -new-instance -no-remote -profile "$profile_dir" -url http://locahost:10080
-    open "$binary" -n --args -new-instance -no-remote -profile "$profile_dir" -url http://locahost:10080
+    echo " " $open -n --args -new-instance -no-remote -profile "$profile_dir" -url http://locahost:10080
+    $open -n --args -new-instance -no-remote -profile "$profile_dir" -url http://locahost:10080
```

Comment:
You should update your repository – it looks like you are working from an old revision.  In particular this has the typo of `locahost` instead of `localhost` which was fixed a while ago (and gives this PR a merge conflict)